.buildkite/lint

```diff
@@ -36,6 +36,7 @@ if [ -n "${SSH_DEPLOY_PRIVKEY:-}" ]; then
 	mkdir -m 0700 -p $HOME/.ssh
 	echo "$SSH_DEPLOY_PRIVKEY" >$HOME/.ssh/id_ed25519
 	chmod 600 $HOME/.ssh/id_ed25519
+
```

Comment:
I think the whole commit can be dropped. It certainly doesn't do what it claims to do.

Reply:
Good catch. That was to trigger a test build indeed.